src/bin/index.ts

```diff
@@ -36,6 +39,36 @@ const parseCommandLineArguments = () => {
           defaultDescription: "The current working directory",
         })
       )
+      .command(Commands.New, "Creates a new CDK stack", (yargs) =>
+        yargs
+          .option("multi-app", {
+            type: "boolean",
+            description:
+              "Create the stack files within sub directories as the project defines multiple apps (defaults to false)",
+            default: false,
+          })
+          .option("init", {
+            type: "boolean",
+            description: "Create the cdk directory before building the app and stack files (defaults to true)",
+            default: true,
```

Comment:
For a clearer API, I'd suggest this be `no-init` and default it to false, as we're then effectively using the _presence_ of the flag as the value.

This means we go from:

```
npx @guardian/cdk@latest new --init false ...
```

To:

```
npx @guardian/cdk@latest new --no-init
```

Reply:
TIL yargs treats `no-` as the inverse of a boolean flag! So please ignore this suggestion.